ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-commands.sh

```diff
@@ -167,6 +177,16 @@ if [[ "${CLUSTER_TYPE}" == gcp ]]; then
     popd
 fi
 
+if [[ "${CLUSTER_TYPE}" == alibabacloud ]]; then
+    pushd /tmp
+    curl -O https://aliyuncli.alicdn.com/aliyun-cli-linux-3.0.83-amd64.tgz 
+    tar -xzf aliyun-cli-linux-3.0.83-amd64.tgz
+    mkdir -p bin/
+    mv aliyun bin/
```

Comment:
we should drop this until we find a test-case that actually needs the `aliyun` binary in the `PATH`.

Reply:
Will remove.